File: Front/nexta/src/models/BasketProduct.ts
export interface BasketProduct{
    userId: string,
    productId: string,
    count: number,
    deliveryDate: string,
    article:string,
    name:string,
    status: BasketProductStatus,
    newPrice:number,
    oldPrice:number
}

export enum BasketProductStatus{
    Unknown = -1, //Неизвестный статус
    Accepted = 0, //Принят
    AtWork = 1, //В работе
    Rejected = 2, //Отказ
    Waiting = 3 //Ожидает
}

File: Back/Nexta/Nexta.Domain/Models/BasketProduct.cs
using Nexta.Domain.Enums;

namespace Nexta.Domain.Models
{
    public class BasketProduct
    {
        public Guid UserId { get; set; }
        public Guid ProductId { get; set; }
        public Product? Product { get; set; }
        public int? Count { get; set; }
        public DateOnly? DeliveryDate { get; set; }
        public BasketProductStatus? Status { get; set; }
    }
}
